packages/engine/src/worker/runner/javascript/mod.rs

```diff
@@ -2011,6 +1993,22 @@ fn new_js_string<'s>(
     v8::String::new(scope, s).expect(&format!("Could not create JS String: {s}"))
 }
 
+/// Helper function to get the exception from a [`v8::TryCatch`]
+fn try_catch_exception<'s, 'p: 's, S>(try_catch: &mut v8::TryCatch<'s, S>) -> Option<String>
+where
+    v8::TryCatch<'s, S>: AsMut<v8::HandleScope<'p, ()>>,
+    v8::TryCatch<'s, S>: AsMut<v8::HandleScope<'p, v8::Context>>,
+    v8::TryCatch<'s, S>: AsMut<v8::Isolate>,
```

Comment:
Why do we need `AsMut<Isolate>`?

Reply:
`to_rust_string_lossy` takes a `&mut Isolate` so I thought the bound was needed. But it's implemented for all generics so no need.
Removed in https://github.com/hashintel/hash/pull/493/commits/eb94d5c347065c16fc4200a4c898028f747f6aba